<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>854</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="windowWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="notebookManagerWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QWidget" name="notebookManagerHeaderWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <widget class="QWidget" name="notebookManagerTitleWidget" native="true">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>0</y>
            <width>281</width>
            <height>41</height>
           </rect>
          </property>
          <widget class="QLabel" name="notebookManagerTitleLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>77</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="indent">
            <number>1</number>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="treeWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QTreeWidget" name="QtreeWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>511</width>
            <height>251</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
         <widget class="QWidget" name="notebookActionWidget" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>300</y>
            <width>511</width>
            <height>71</height>
           </rect>
          </property>
          <widget class="QPushButton" name="notebookActionButton">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>30</y>
             <width>89</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
          <widget class="QLabel" name="notebookActionLabel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>261</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="notebookAction_lineEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>411</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="noteActionWidget" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>420</y>
            <width>511</width>
            <height>351</height>
           </rect>
          </property>
          <widget class="QLineEdit" name="noteAction_lineEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>511</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextEdit" name="noteAction_textEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>180</y>
             <width>511</width>
             <height>111</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="noteAction_comboBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>100</y>
             <width>511</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="noteActionButton">
           <property name="geometry">
            <rect>
             <x>400</x>
             <y>300</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
          <widget class="QLabel" name="noteActionLabel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>261</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="noteActionSelectorLabel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>80</y>
             <width>261</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="noteActionDescriptionLabel">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>160</y>
             <width>261</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>280</y>
            <width>511</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>390</y>
            <width>731</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
